services/backend/.env.example

```diff
@@ -13,4 +13,5 @@ ADMIN_DEFAULT_PASSWORD=password
 
 DB_CONNECTION={"dbname":"backend","username":"backend","password":"backend","host":"db","port":5432}
 
+LOCALSTACK_URL=http://localstack:4566
```

Comment:
This should be named `AWS_ENDPOINT_URL` . Localstack is only a mock environment and in context of the app it does not exist. The app still thinks it uses a regular AWS.